api/client.go

```diff
@@ -321,3 +320,19 @@ func partialText(body []byte, limit int) string {
 
 	return string(body[:limit]) + "..."
 }
+
+func HandleConnError(err error) error {
```

Comment:
Would be great to have a godoc comment on this since it's exported:

```suggestion
// HandleConnError translates common connection errors into more informative human
// readable errors. Returns `nil` if the error was not handled, so it is the callers responsibility to
// return the original error if `HandleConnError` returns nil.
func HandleConnError(err error) error {
```

In other places our `handleErr` function returns the original error if it doesn't find any special cases, so we should be clear that this one returns `nil` if it doesn't handle anything.